### PR TITLE
Have fixed some errors

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
these will help others more

NOTE: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributing

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -889,3 +893,6 @@ end
 ```
 
 [Back to code style TOC](#table-of-contents---code-style)
+[Back to TOC](#table-of-contents)
```
Comment (hishamhm) [inline CONTRIBUTING.md]: This one already has a back link next to it — could you remove this one?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ Consult the Table of Contents below, and jump to the desired section.
 
 If you are a Kong Enterprise customer, contact the Enterprise Support channels
 by opening an Enterprise support ticket on
-[https://support.konghq.com](https://support.konghq.com/).
+[support](https://support.konghq.com/).
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: This change feels unnecessary and doesn't really produce a readable sentence.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ Consult the Table of Contents below, and jump to the desired section.
...
 
 If you are experiencing a P1 issue, please call the [24/7 Enterprise Support
 phone line](https://support.konghq.com/hc/en-us/articles/115004921808-Telephone-Support)
 for immediate assistance, as published in the Customer Success Reference Guide.
 
 If you are interested in becoming a Kong Enterprise customer, please visit
-https://konghq.com/kong-enterprise-edition/ or contact us at
+[https://konghq.com/kong-enterprise-edition/](https://konghq.com/kong-enterprise-edition/) or contact us at
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: This change is unnecessary, GitHub's markdown parser will already produce a link with the previous version. Also, this makes the line go beyond the 80 columns limit enforced in this document.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ make without coding:
 - Reporting a bug (see the [report bugs](#where-to-report-bugs) section)
 - Helping other members of the community on the support channels
 - Fixing a typo in the code
-- Fixing a typo in the documentation at https://docs.konghq.com (see
+- Fixing a typo in the documentation at [https://docs.konghq.com](https://docs.konghq.com) (see
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: ditto (unneccesary + columns limit).

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -271,7 +271,7 @@ Here is a template of what your commit message should look like:
 <BLANK LINE>
 <footer>
 ```
-
+[Back to TOC](#table-of-contents)
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: style: note that 2 line breaks are enforced in this file before each header. There are other changes in this patch that do not respect this rule.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -293,6 +293,7 @@ accepted types are:
   considered part of a refactor, build process updates, dependency bumps, or
   auxiliary tools and libraries updates (LuaRocks, Travis-ci, etc...).
 
+[Back to TOC](#table-of-contents)
```
Comment (thibaultcha) [inline CONTRIBUTING.md]: Note that "Back to TOC" links were not added in this section _on purpose_ (within `<h5>` tags, that is).

**Comment by thibaultcha** [comment]
I double checked, and none of the TOC links are missing. When ommitted, it is intentional. The other changes aren't helping in clarifying the content, sorry. I just pushed https://github.com/Kong/kong/commit/88a530c68e09e0bde74a1904ee908054718f6195 which will help avoiding future contributors getting confused with the exotic styling in this file.

Closing this PR now, thank you for your time on this and trying to improve the project!

File: CONTRIBUTING.md (hunk 1 above)
Comment (sakshamtaneja21) [reply CONTRIBUTING.md]: sure

File: CONTRIBUTING.md (hunk 4 above)
Comment (sakshamtaneja21) [reply CONTRIBUTING.md]: ok ill change

**Pull request closed without merging**